public/app/themes/justice/inc/templates.php

```diff
@@ -61,6 +62,7 @@ public function replaceWordpressBlocks(string $block_content, array $block): str
             switch ($block['blockName']) {
                 case 'core/paragraph':
                 case 'core/table':
+                case 'moj/to-the-top':
```

Comment:
I'm thinking this loop might get a bit slow for pages with 100s of blocks.

WordPress by default doesn't loop over blocks in this way - usually what is in the database is very close to the final rendered output. 

As `moj/to-the-top` will always have the same content do we need to run it through `renderLinks`?

In the case of `moj/to-the-top` could we simplify things by doing the equivalent of `$block_content = <use the to-the-top template with these static properties>`?